### ISSUE TITLE
workspaces can be non numerical

### ISSUE THREAD

**Issue by mrVanDalo** [tmfink/i3-wk-switch#13]
nice script (also in the process of migrating from xmonad to i3).

I'm using named workspaces and would like to use your script with named workspaces as well.

`i3-msg workspace "hello world"` works (maybe not back when you created the script)
but `i3-wk-switch "hello world"` does not work :(

Here is the script I would like to use 

```shell
set -e
set -o pipefail

i3-msg -t get_workspaces | \
jq --raw-output '.[] | .name' | \
rofi -dmenu -p 'Select Workspace' | \
while read workspace
do
  i3-wk-switch "$workspace"
done
```

Maybe when I have some time in the next weeks, I'll write a pull-request if that is ok.

**Comment by tmfink** [comment]
Good catch--I opened PR #14.

**Comment by tmfink** [comment]
Fixed via #14
